projects/packages/videopress/src/class-initializer.php

```diff
@@ -228,6 +228,14 @@ public static function register_videopress_video_block() {
 			)
 		);
 
+		wp_enqueue_script(
+			self::JETPACK_VIDEOPRESS_VIDEO_VIEW_HANDLER . '-iframe-api',
+			'https://wordpress.com/wp-content/plugins/video/assets/js/videojs/videopress-iframe-api.js',
+			array(),
+			'1.0.0',
```

Comment:
Would it be worth updating this version number to something that would change every week, like we do for other resources loaded from WordPress.com? Something like for a flush every week for example:

```suggestion
			gmdate( 'YW' ),
```

I guess that depends how often this file will be changed.

Reply:
Yes, I think that lib won't be updated very often, but still. Thanks